packages/appium/test/unit/config.spec.js

```diff
@@ -214,18 +214,21 @@ describe('Config', function () {
     });
   });
 
-  describe('validateTmpDir', function () {
-    it('should fail to use a tmp dir with incorrect permissions', function () {
-      validateTmpDir('/private/if_you_run_with_sudo_this_wont_fail').should.be.rejectedWith(
-        /could not ensure/
+  describe('requireDir', function () {
+    it('should fail to use a dir with incorrect permissions', function () {
+      requireDir('/private/if_you_run_with_sudo_this_wont_fail').should.be.rejectedWith(
```

Comment:
Well this is interesting. I had run unit tests on my local Windows machine recently, and a few days later discovered an empty directory at `C:\private\if_you_run_with_sudo_this_wont_fail`. I assume these tests were meant to be run on Unix machines, so I wonder if this behavior is expected?

Reply:
yes, they are optimized for macOS
we tried to run them different platforms but never had time to stabilize 
Also on Windows they seem to be times slower 🤷